doc/developer-guide/deployment/nginx.rst

```diff
@@ -10,20 +10,53 @@ running on your system.
 When proxying, don't forget to check the config files of the sites you
 are planning to server (the ``user/sites/your_site/config``
 files). The ``hostname`` value should not contain any port number, if
-you run from port 80: ``{hostname, "test.zotonic.com"}``.
+you run from port 80/443: ``{hostname, "test.zotonic.com"}``.
 
-Zotonic also makes use of a websocket connection for MQTT messages
-under the ``/mqtt-transport`` location and so you have to add an extra
-proxy directive for this.
+Zotonic configuration
+---------------------
+
+Example of ``~/.zotonic/[release]/zotonic.config`` ip/port settings when
+terminating SSL in nginx proxy and using plain HTTP towards the Zotonic
+backend:
+
+.. code-block:: erlang
+
+  %%% IP address on which Zotonic will listen for HTTP requests.
+  {listen_ip, any},
+
+  %%% Port on which Zotonic will listen for HTTP requests.
+  {listen_port, 8000},
+
+  %%% Port on which Zotonic will listen for HTTPS requests.
+  %%% Set to the atom 'none' to disable SSL
+  {ssl_listen_port, none},
 
-Below is a configuration file we use to proxy nginx to zotonic. Be
+  %%% Outside port on which Zotonic will listen for HTTP requests.
+  {port, 80},
+
+  %%% Outside port zotonic uses to receive incoming HTTPS requests.
+  {ssl_port, 443},
+
+  %%% Force sites to use SSL. Redirects http requests to https.
+  {ssl_only, true},
```

Comment:
On master we now always redirect to ssl, so we need to remove this setting.

Reply:
Checked the source code, we have removed it already, so we can also remove it from the docs here.

Reply:
Will remove it from the other docs as well